pint/models/dispersion_model.py

```diff
@@ -258,4 +309,6 @@ def print_par(self,):
             result += getattr(self, DMX_mapping[ii]).as_parfile_line()
             result += getattr(self, DMXR1_mapping[ii]).as_parfile_line()
             result += getattr(self, DMXR2_mapping[ii]).as_parfile_line()
+        result += getattr(self, 'SWM').as_parfile_line()
```

Comment:
This was added because test_parfile_writing.py has a DMX model with SWM and DMX write does not write SWM information by default. I looked at the behaviour of Tempo2, and it looks like it writes SWM parameters even when they haven't been explicitly set by the user, because they are always used in delay calculations.